torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,116 @@
+import dataclasses
+import io
+from enum import Enum
+from typing import Optional, Protocol, Union
+
+import torch
+
+
+def dynamo_export(
+    model: torch.nn.Module,
+    /,
+    *model_args,
+    export_options: Optional[ExportOptions] = None,
+    **model_kwargs,
+) -> ExportOutput:
+    """Export a torch.nn.Module to a ONNX graph
+
+    Args:
+        model: Model to be exported
+        model_args: Positional inputs to `torch_model`
+        export_options: Export options
+        model_kwargs: Keyword inputs to `torch_model`
+
+    Returns:
+        An in-memory representation of the exported ONNX model.
+        See ``ExportOutput`` for reference.
+    """
+    ...
+
+
+class ExportLogLevel(Enum):
+    """Class for choosing log level.
+
+    Names and values follows https://docs.python.org/3/library/logging.html#levels"""
+
+    CRITICAL = 50
+    ERROR = 40
+    WARNING = 30
+    INFO = 20
+    DEBUG = 10
+    NOTSET = 0
+
+
+@dataclasses.dataclass
+class ExportOptions:
+    """Class for keeping track of ONNX exporter flags can perform."""
```

Comment:
```suggestion
    """Class for keeping track of ONNX exporter flags."""
```

Or `ONNX exporter options.`?